blazor/media-query/getting-started.md

```diff
@@ -91,56 +90,28 @@ await builder.Build().RunAsync();
 {% endhighlight %}
 {% endtabs %}
 
-## Add script reference
+## Add stylesheet and script resources
```

Comment:
Is script necessary on this component?

Reply:
No, Script wasn't necessary so removed from this PR.